### PR TITLE
changed dialogs to bottom sheet fragments

### PR DESCRIPTION
**Pull request by timi10x**

### CODE REVIEW

File: app/src/main/java/com/example/awesomefish/ui/GameLauncher.kt [hunk 1]
```diff
@@ -30,9 +30,7 @@ class GameLauncher(context: Context, soundManager: SoundManager) : View(context)
         return SceneManager.get().onTouch(event)
     }
 
-    fun onPause() {
```
Comment (AliyuAbdullahi) [inline app/src/main/java/com/example/awesomefish/ui/GameLauncher.kt]: do we need this space?

File: app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+private const val HIGH_SCORE_TAG = "highScore"
+
+class HighScoreBottomSheet : RoundedBottomSheetDialogFragment() {
+
+    private val repository = DI.provideScoreRepository()
+    lateinit var emptyText: TextView
```
Comment (AliyuAbdullahi) [inline app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt]: can emptyText be private?

File: app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        return inflater.inflate(R.layout.highscore_bottom_sheet, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        emptyText = view.findViewById(R.id.empty_text)
+        GlobalScope.launch {
+            withContext(Dispatchers.Main) {
+                val scores =
+                    repository.getAll().sortedBy { it.scoreValue }
```
Comment (AliyuAbdullahi) [inline app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt]: while we are here, lets' make this one liner

File: app/src/main/java/com/example/awesomefish/ui/dialogs/PauseMenuDialog.kt [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+import com.example.awesomefish.R
+import com.example.awesomefish.shared.FontManager
+import com.example.awesomefish.ui.menu.PAUSE_MENU_TAG
+import com.example.awesomefish.ui.menu.PauseMenu
+import com.example.awesomefish.utils.RoundedBottomSheetDialogFragment
+import kotlinx.android.synthetic.main.pause_menu.*
+
+class PauseMenuDialog : RoundedBottomSheetDialogFragment() {
+
+    lateinit var resume: TextView
```
Comment (AliyuAbdullahi) [inline app/src/main/java/com/example/awesomefish/ui/dialogs/PauseMenuDialog.kt]: can all these fields be private?

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -8,4 +8,19 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <!-- **************set the rounded drawable as background to your bottom sheet************** -->
```
Comment (AliyuAbdullahi) [inline app/src/main/res/values/styles.xml]: Let's add space between the last * and **set** and same with **sheet**. Something like ** set the... bottom **.
Maybe two spaces. Same thing with the bottom as well.

**Review by AliyuAbdullahi — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/awesomefish/ui/GameLauncher.kt (hunk 1 above)
Comment (timi10x) [reply app/src/main/java/com/example/awesomefish/ui/GameLauncher.kt]: oh didn't see that tho
would fix it

File: app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt (hunk 2 above)
Comment (timi10x) [reply app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt]: definitely, would add that to it

File: app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt (hunk 3 above)
Comment (timi10x) [reply app/src/main/java/com/example/awesomefish/ui/dialogs/HighScoreBottomSheet.kt]: alright

File: app/src/main/java/com/example/awesomefish/ui/dialogs/PauseMenuDialog.kt (hunk 4 above)
Comment (timi10x) [reply app/src/main/java/com/example/awesomefish/ui/dialogs/PauseMenuDialog.kt]: sure

File: app/src/main/res/values/styles.xml (hunk 5 above)
Comment (timi10x) [reply app/src/main/res/values/styles.xml]: okay i see that now
would do it right now

**Pull request merged**